### PR TITLE
MM-13368 Do not truncate tall images in preview screen

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Respect the viewport height to display the image preview

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-13368

### CODE REVIEW

File: app/utils/images.js [hunk 1]
```diff
@@ -34,6 +34,9 @@ export const calculateDimensions = (height, width, viewPortWidth = 0, viewPortHe
     ) {
         imageHeight = IMAGE_MIN_DIMENSION;
         imageWidth = imageHeight * heightRatio;
+    } else if (viewPortHeight && imageHeight > viewPortHeight) {
```
Comment (saturninoabril) [inline app/utils/images.js]: Can we add test case to cover this?

Comment (enahum) [reply app/utils/images.js]: sure

Comment (enahum) [reply app/utils/images.js]: this is already covered by the test cases

Comment (enahum) [reply app/utils/images.js]: added

File: app/utils/images.test.js [hunk 2]
```diff
@@ -62,4 +62,9 @@ describe('Images calculateDimensions', () => {
         expect(height).toEqual(45);
         expect(width).toEqual(310);
     });
+
+    it('Set the viewPort height defined should return the image capped to the viewpoort', () => {
```
Comment (saturninoabril) [inline app/utils/images.test.js]: spelling check: `viewpoort`

Comment (enahum) [reply app/utils/images.test.js]: @saturninoabril thanks typo fixed

**Pull request merged**